stress/shell/src/main/java/alluxio/stress/cli/StressJobServiceBench.java

```diff
@@ -0,0 +1,280 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.cli;
+
+import alluxio.AlluxioURI;
+import alluxio.ClientContext;
+import alluxio.cli.fs.command.DistributedLoadCommand;
+import alluxio.cli.fs.command.DistributedLoadUtils;
+import alluxio.client.file.FileOutStream;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.FileSystemContext;
+import alluxio.client.job.JobMasterClient;
+import alluxio.conf.InstancedConfiguration;
+import alluxio.exception.AlluxioException;
+import alluxio.grpc.CreateFilePOptions;
+import alluxio.grpc.DeletePOptions;
+import alluxio.grpc.WritePType;
+import alluxio.stress.BaseParameters;
+import alluxio.stress.StressConstants;
+import alluxio.stress.jobservice.JobServiceBenchParameters;
+import alluxio.stress.jobservice.JobServiceBenchTaskResult;
+import alluxio.stress.jobservice.JobServiceBenchTaskResultStatistics;
+import alluxio.util.CommonUtils;
+import alluxio.util.ConfigurationUtils;
+import alluxio.util.FormatUtils;
+import alluxio.util.executor.ExecutorServiceFactories;
+import alluxio.worker.job.JobMasterClientContext;
+
+import com.beust.jcommander.ParametersDelegate;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.HdrHistogram.Histogram;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Job Service stress bench.
+ */
+public class StressJobServiceBench extends Benchmark<JobServiceBenchTaskResult> {
+  private static final Logger LOG = LoggerFactory.getLogger(StressJobServiceBench.class);
+  public static final int MAX_RESPONSE_TIME_BUCKET_INDEX = 0;
+  @ParametersDelegate
+  private JobServiceBenchParameters mParameters = new JobServiceBenchParameters();
+
+  /**
+   * Creates instance.
+   */
+  public StressJobServiceBench() {}
+
+  /**
+   * @param args command-line arguments
+   */
+  public static void main(String[] args) {
+    mainInternal(args, new StressJobServiceBench());
+  }
+
+  @Override
+  public void prepare() throws Exception {
```

Comment:
so if i understood this correctly, this prepare is going to run on every job worker, and creating potentially duplicate files if it is a shared file system right?

Reply:
can this be avoided?

Reply:
in line 96, every worker would try to create its own directory with mBaseParameters.mId, so I think it would be different directories(directories contains the same files)

Reply:
i see. what does `mBaseParameters.mId` return when we are not using cluster mode?  is it still valid?

Reply:
it's valid, in my local machine it's local-task-0

Reply:
Did you see a "local-task-0" directory in cluster mode in your test? If yes, I'm more towards not creating this folder because it could be confusing for people though not affecting performance.